graph-db/serverless/src/utils/aws/getSqsConfig.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Returns an environment specific configuration object for Lambda
```

Comment:
should this docstring and the @return be 

```suggestion
 * Returns an environment specific configuration object for SQS
```